frontend/src/app/versions.tsx

```diff
@@ -372,7 +372,7 @@ class Versions extends Component<IProps, any> {
                         const draftTag = responseJSON.draft;
                         const versionCount = responseJSON.__children__.length
 
-                        for (let i = versionCount - 1; i > versionCount - 3 && i >= 0; i--) {
+                        for (let i = versionCount - 1; i >= 0; i--) {
```

Comment:
What is the rationale for removing the condition here? Especially because this PR is for search results, and versions.tsx is unrelated to that.